scripts/testing/check-container-age.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+# container image name, eg. "quay.io/rhinstaller/anaconda-ci:master"
+container_name="$1"
+# when is an image too old - string for `date --date=`
+too_old_description="60 hours ago"
+
+# dates in unix epoch seconds
+ts_container_created=$(date --date="$(podman image inspect --format '{{.Created}}' $container_name | awk '{print($1 " " $2)}')" +%s)
+ts_too_old=$(date +%s --date="$too_old_description")
+
+if [ "$ts_too_old" -gt "$ts_container_created" ] ; then
+  echo "====================================================================="
+  echo "WARNING: Container is too old: $container_name"
```

Comment:
It might good to add the container age here, something like *WARNING: Container is too old: $container_name ($age)*

Reply:
Unfortunately this is almost impossible. Apparently I would have to calculate this myself, or install something.